lib/rubocop-rspec.rb

```diff
@@ -9,21 +9,23 @@
 require_relative 'rubocop/rspec/version'
 require_relative 'rubocop/rspec/inject'
 require_relative 'rubocop/rspec/node'
-require_relative 'rubocop/rspec/top_level_describe'
 require_relative 'rubocop/rspec/wording'
 require_relative 'rubocop/rspec/language'
 require_relative 'rubocop/rspec/language/node_pattern'
-require_relative 'rubocop/rspec/top_level_group'
+
+require_relative 'rubocop/rspec/mixin/align_let_brace'
+require_relative 'rubocop/rspec/mixin/top_level_describe'
+require_relative 'rubocop/rspec/mixin/top_level_group'
+require_relative 'rubocop/rspec/mixin/variable'
+require_relative 'rubocop/rspec/mixin/final_end_location'
+require_relative 'rubocop/rspec/mixin/blank_line_separation'
```

Comment:
I guess maybe `rubocop/rspec/cop/mixin/*` if they are intended to be included in cops like [RuboCop does it](https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop.rb#L51)?

Reply:
@pirj yeah, I thought about that briefly. The weird thing in our case is that `FinalEndLocation` gets included in two places, one being `lib/rubocop/rspec/corrector/move_node.rb`. Should I put that in the `cop/mixin/` directory anyway, or leave it in `rubocop/rspec/`? It also gets included in `blank_line_separation.rb` which is itself included in cops.

Reply:
@pirj I went ahead and moved them over to the `cop/rspec/` folder. That one case is a little odd, but I think it's probably still better this way.